pom.xml

```diff
@@ -57,7 +57,8 @@
                             SLING-INF/oak:index;overwrite:=true;uninstall:=true;path:=/oak:index
                         ]]></Sling-Initial-Content>
                         <Sling-Model-Packages>
-                            com.redhat.pantheon.model
+                            com.redhat.pantheon.model,
+                            com.redhat.pantheon.use
```

Comment:
Mmm... lets talk about our package structure. We need to start thinking about how we want to structure our code... there are different options.

Reply:
Sure, open to suggestions. There is also a `<Sling-Model-Classes>` tag we can use to be more granular.